frame/support/src/wasm_tracing.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! # To allow tracing in WASM execution environment
+//!
+//! Facilitated by `sp_io::wasm_tracing`
+
+/// Indicates whether to run traces in wasm
+pub static mut WASM_TRACING_ENABLED: bool = true;
```

Comment:
This variable it completely useless, we still call into the host every time.

Reply:
yes, sorry, my mistake - last commit was bad - just reverted it